.github/workflows/conformance-ginkgo.yaml

```diff
@@ -300,7 +300,7 @@ jobs:
           cmd: |
             git config --global --add safe.directory /host
             mv /host/helm /usr/bin
-            mv /host/cilium-cli /usr/bin
+            mv /host/cilium /usr/bin/cilium-cli
```

Comment:
looking at recent failures with scheduled runs (like [this one](https://github.com/cilium/cilium/actions/runs/9782530035)), it doesn't look like sysdump is being collected. @aanm did we accidentally stop running `cilium sysdump` command with this pull request? https://github.com/cilium/cilium/pull/26143

Reply:
@michi-covalent I'm not sure if I follow. We do have sysdumps on some test failures, no?

Reply:
i only checked like 2 archives from failed runs so maybe, but i couldn't find any step that runs the sysdump command in https://github.com/cilium/cilium/blob/main/.github/workflows/conformance-ginkgo.yaml 💭